gravitee-apim-cypress/cypress/fixtures/fakeData/users/users.ts

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { BasicAuthentication } from '../../model/users';
+import { BasicAuthentication } from 'model/users';
```

Comment:
sorry but cannot add a comment about `setData`
`setData` does not mean anything in my opinion
please rename this folder to `static` or `json`

Reply:
Ok, as discussed, I'll change it back to 'fakers' and name the other one 'json'.